test/images/sample-device-plugin/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM scratch
```

Comment:
oh, this fails because it doesn't have a `/tmp` directory:

```
$ docker run --rm -ti gcr.io/kubernetes-e2e-test-images/sample-device-plugin-amd64:1.0
I0130 19:34:13.776201       1 sampledeviceplugin.go:82] pluginSocksDir:
log: exiting because of error: log: cannot create log: open /tmp/sampledeviceplugin.1535d230397a.unknownuser.log.INFO.20190130-193413.1: no such file or directory
```

If we're feeling adventurous, we could try using `gcr.io/distroless/static:latest` here.